README.md

```diff
@@ -1,11 +1,19 @@
-# Contract level buyer information
+# Multiple buyers information
```

Comment:
We already allow multiple buyers in plain OCDS (at the tender stage), so we need to distinguish this more clearly. I don't know a succinct way, so I'm suggesting this for now.

```suggestion
# Buyer per award or contract
```